operators/cmd/manager/main.go

```diff
@@ -213,10 +213,7 @@ func execute() {
 		Namespace: viper.GetString(NamespaceFlagName),
 	}
 	metricsPort := viper.GetInt(MetricsPortFlag)
-	if metricsPort != 0 {
-		opts.MetricsBindAddress = fmt.Sprintf(":%d", metricsPort)
-		log.Info(fmt.Sprintf("Exposing Prometheus metrics on /metrics%s", opts.MetricsBindAddress))
-	}
+	log.Info("Exposing Prometheus metrics on /metrics", "port", metricsPort)
```

Comment:
Looks like this is changing the behaviour?

Reply:
Ah you're right, this was sloppy. That said I wonder if we do just want to set a default port (probably 80?) and always expose prometheus metrics